docs/content/installation/advanced_configuration/fds_mode.md

```diff
@@ -126,14 +254,22 @@ spec:
     # WHITELIST is the default value
     fdsMode: WHITELIST
     fdsOptions:
-      # set to false to disable automated GraphQL schema generation as part of FDS.
+      # Enterprise-only: set to false to disable automated GraphQL schema generation as part of FDS.
       # true is the default value (enabled)
       graphqlEnabled: true
-{{< /highlight >}}
+```
+{{% /tab %}} 
+{{< /tabs >}}
 
----
+### FDS modes
 
-## Blacklisting Namespaces & Upstreams
+FDS can run in one of 3 modes:
+
+* `BLACKLIST`: The most liberal FDS polling policy. Using this mode, FDS will poll any service unless its namespace or the service itself is explicitly blacklisted.
+* `WHITELIST`: A more restricted FDS polling policy. Using this mode, FDS will poll only those services who either live in an explicitly whitelisted namespace, or themselves are are explicitly whitelisted. *`WHITELIST` is the default mode for FDS*.
+* `DISABLED`: FDS will not run. **Upstream Discovery Service** (UDS) will still run as normal.
```

Comment:
i think this was existing content, but just while we're here. feel free to change
```suggestion
* `BLACKLIST`: The most liberal FDS polling policy. FDS polls all services, unless the service or the service's namespace is blacklisted.
* `WHITELIST` (default mode): A more restrictive FDS polling policy. FDS polls only services that are either whitelisted or exist in a whitelisted namespace.
* `DISABLED`: FDS does not run. UDS continues to run as normal.
```